packages/dashboard/src/components/account-button-toggle/accounts-button-toggle.scss

```diff
@@ -0,0 +1,31 @@
+.button-toggle {
+    &__container {
+        width: 20.8rem;
+        height: 4.8rem;
+        margin: 3rem auto;
+        padding: 0.8rem;
+        background-color: var(--general-section-1);
+        border-radius: 0.8rem;
+    }
+    &__items {
+        .dc-button-menu__wrapper {
```

Comment:
```suggestion
        .dc-button-menu {
           &__wrapper {
```

Reply:
I think it's better to use sass notations here as well since we have many sub selectors